modules/adding-an-odh-instance.adoc

```diff
@@ -0,0 +1,36 @@
+:_module-type: PROCEDURE
+//pv2hash: 
+
+[id='adding-an-odh-instance_{context}']
+= Adding an Open Data Hub instance 
+
+[role='_abstract']
+Add an Open Data Hub instance to your project to be able to give access to an Open Data Hub dashboard.
```

Comment:
Why does the user have to add an ODH instance to their project? What does this provide?

Reply:
By adding an Open Data Hub instance to your project, you can access the URL for your Open Data Hub dashboard and share it with data science users.